cmd/utils/flags.go

```diff
@@ -1183,6 +1203,18 @@ func setHTTP(ctx *cli.Context, cfg *node.Config) {
 		rpc.ConcurrencyLimit = ctx.GlobalInt(RPCConcurrencyLimit.Name)
 		logger.Info("Set the concurrency limit of RPC-HTTP server", "limit", rpc.ConcurrencyLimit)
 	}
+	if ctx.GlobalIsSet(RPCReadTimeout.Name) {
+		cfg.HTTPTimeouts.ReadTimeout = time.Duration(ctx.GlobalInt(RPCReadTimeout.Name)) * time.Second
+	}
+	if ctx.GlobalIsSet(RPCWriteTimeoutFlag.Name) {
+		cfg.HTTPTimeouts.WriteTimeout = time.Duration(ctx.GlobalInt(RPCWriteTimeoutFlag.Name)) * time.Second
+	}
+	if ctx.GlobalIsSet(RPCIdleTimeoutFlag.Name) {
+		cfg.HTTPTimeouts.IdleTimeout = time.Duration(ctx.GlobalInt(RPCIdleTimeoutFlag.Name)) * time.Second
+	}
+	if ctx.GlobalIsSet(RPCExecutionTimeoutFlag.Name) {
+		cfg.HTTPTimeouts.ExecutionTimeout = time.Duration(ctx.GlobalInt(RPCExecutionTimeoutFlag.Name)) * time.Second
+	}
```

Comment:
@mckim19 
Minor thing but, we need to consider about unit of the value.
In the flag description, the unit of default value is nanosecond. (e.g. `30000000000`, 1 second == 1e+9 nanosecond)

This can lead to confusion for the user as to which unit the value should be written in.

```
  --rpcexecutiontimeout value       HTTP-RPC server execution timeout (default: 30000000000)
  --rpcidletimeout value            HTTP-RPC server idle timeout (default: 120000000000)
  --rpcreadtimeout value            HTTP-RPC server read timeout (default: 30000000000)
  --rpcwritetimeout value           HTTP-RPC server write timeout (default: 30000000000)
```
cc. @jeongkyun-oh

Reply:
thank you so much.  
Im gonna change the flag description. :)

Reply:
@mckim19 
Thanks for changing.
It looks great now :)

```
  --rpcexecutiontimeout value       HTTP-RPC server execution timeout (seconds) (default: 30)
  --rpcidletimeout value            HTTP-RPC server idle timeout (seconds) (default: 120)
  --rpcreadtimeout value            HTTP-RPC server read timeout (seconds) (default: 30)
  --rpcwritetimeout value           HTTP-RPC server write timeout (seconds) (default: 30)
```